### PR TITLE
Update locales.lua

### PR DESCRIPTION
**Pull request by stoneszn**
Added Spanish locale

### CODE REVIEW

File: locales/locales.lua [hunk 1]
```diff
@@ -120,5 +120,61 @@ Locales = {
...
+        ['melee'] = "Pelea en progreso",
+        ['driveby'] = "Disparos desde vehículos en movimiento",
+        ['speeding'] = "Conducción temeraria",
+        ['autotheft'] = "Robo de un vehículo de motor",
+        ['persondown'] = "La persona esta lesionada",
+        ['storerobbery'] = "Robo de tienda",
+        ['fleecabank'] = "Robo del banco Fleeca",
+        ['paletobank'] = "Robo del Banco Paleto",
+        ['pacificbank'] = "Robo del Banco del Pacífico",
+        ['prisonbreak'] = "Prison Break en progreso",
```
Comment (MonkeyWhisper) [inline locales/locales.lua]: This is English, revise to Spansih. Something along the lines of Escape de Carcel

File: locales/locales.lua [hunk 2]
```diff
@@ -120,5 +120,61 @@ Locales = {
...
+        ['vangelico'] = "Robo vangelico",
```
Comment (MonkeyWhisper) [inline locales/locales.lua]: Revise to Robo de Joyeria

File: locales/locales.lua [hunk 3]
```diff
@@ -120,5 +120,61 @@ Locales = {
...
+        ['houserobbery'] = "Robo de casa",
+        ['drugsell'] = "Transferencia sospechosa",
```
Comment (MonkeyWhisper) [inline locales/locales.lua]: Posibles Drogas or something alone those lines

File: locales/locales.lua [hunk 4]
```diff
@@ -120,5 +120,61 @@ Locales = {
...
+        ['carjacking'] = "Levantamiento de autos", 
+        ['vehicletheft'] = "Robo de vehículos", 
+        ['officerdown'] = "Oficial caido",
```
Comment (MonkeyWhisper) [inline locales/locales.lua]: Socorro Policia or something more urgent

File: locales/locales.lua [hunk 5]
```diff
@@ -120,5 +120,61 @@ Locales = {
...
+        ['emsdown'] = "EMS Caido",
```
Comment (MonkeyWhisper) [inline locales/locales.lua]: Doctor Ayuda or Paramedicos Socorro - something more pressing

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]
Please change requested and I can merge this PR .

**Comment by stoneszn** [comment]
> Transferencia sospechosa

Done

**Pull request merged**